File: bms1.py
import sys
import os
from Stream import Stream


def eprint(*args, **kwargs):
    print(*args, file=sys.stderr, **kwargs)


def main():
    if len(sys.argv) != 2:
        eprint("Error:", "Missing source file name!", sep=" ")
        exit(1)

    stream = Stream(sys.argv[1])
    stream.parse_file()


if __name__ == "__main__":
    main()

File: c++/bms1.cpp
//
// Created by odehnaltomas on 24.10.19.
//
#include "Stream.h"

int main(int argc, char **argv) {
	if(argc != 2) {
		printf("ERROR: Input file is missing.");
		exit(1)
	}

	Stream stream = new Stream(argv[1]);
	stream.parse_file()

}